mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -209,6 +215,37 @@ def __statistics_overview(self, series, col):
                     dates.sort_values().iloc[math.floor(len(dates) / 2)].isoformat()
                 )
                 data[f'{col}/min'] = dates.min().isoformat()
+            elif column_type == ColumnType.TEXT:
+                text_series = series_non_null
+                string_length = text_series.str.len()
+                data[f'{col}/avg_string_length'] = string_length.mean()
+                data[f'{col}/min_character_count'] = string_length.min()
+                data[f'{col}/max_character_count'] = string_length.max()
+                text_series = text_series.str.replace(PUNCTUATION, ' ', regex=True)
+                text_series = text_series.str.lower()
+                text_series = text_series.str.split('\s+')
+                text_series = text_series.apply(
+                    lambda words: [word for word in words if word != '']
+                )
+
+                word_count = text_series.map(len)
+                data[f'{col}/max_word_count'] = word_count.max()
+                data[f'{col}/avg_word_count'] = word_count.mean()
+                data[f'{col}/min_word_count'] = word_count.min()
+
+                exploded_text_series = text_series.explode()
+                data[f'{col}/word_distribution'] = (
+                    exploded_text_series.value_counts().head(VALUE_COUNT_LIMIT).to_dict()
+                )
+                data[f'{col}/word_count_excl_stopwords'] = (
```

Comment:
hmm, i was thinking the average word count excluding stop words. total word count is not useful. let's just comment it for now.

Reply:
Commented out, added TODO to later come back and add this statistics back